app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
+    redirect_to root_path
+  end
+
+  def show 
+    @game = Game.find_by_id(params[:id])
```

Comment:
Although it's tempting, I would avoid using variants of `find_by` like `find_by_id` or `find_by_name`. They aren't defined explicitly and instead take advantage of Ruby's process for looking up missing constants, which makes them inefficient. If you're looking things up just based on ID, `find` is what you need, and if you're using a different property, then the syntax is `find_by(property: value)`.